POC/db/models/stock_models/db_models.py

```diff
@@ -0,0 +1,75 @@
+from sqlmodel import SQLModel, Field as SField
+from datetime import datetime
+from typing import Literal, Optional
+from pydantic import BaseModel, Field
+
+
+class DbInfoForm(BaseModel):
```

Comment:
Should the Pydantic BaseModel (Forms) be in the same file as the SQLModel Classes? or in form_models.py?

Reply:
@Zaubeerer